### PR TITLE
Make changes to about-us pages

### PR DESCRIPTION
**Pull request by SmithPeder**
* Create new board type
* Add organisasjon
* Add utnevnelser
* Remove hardcoded categoryOptions from reducers

### CODE REVIEW

**Review by LudvigHz — APPROVED** [review]
LGTM :shipit:

**Review by eikhr — APPROVED** [review]

File: app/routes/pages/PageDetailRoute.js [hunk 1]
```diff
@@ -106,6 +139,14 @@ const sections: {
   },
 };
 
+export const categoryOptions = Object.keys(sections)
```
Comment (odinuge) [inline app/routes/pages/PageDetailRoute.js:142]: :raised_hands:

**Review by odinuge — APPROVED** [review]
Good to see that this generic stuff werkz, at least kinda! Nice :clap:

**Comment by erlingfn** [comment]
This depends on webkom/lego#2840

**Review by LudvigHz — APPROVED** [review]

**Review by eikhr — APPROVED** [review]
LGTM

**Pull request merged**